scripts/components/OpenSearch/install.sh

```diff
@@ -82,12 +87,23 @@ cd $DIR
 MAJOR_VERSION=`echo $VERSION | cut -d. -f1`
 if [ "$DISTRIBUTION" = "tar" ]; then
     cp -v ../../../scripts/startup/tar/linux/opensearch-tar-install.sh "$OUTPUT/"
-elif [ "$DISTRIBUTION" = "deb" -o "$DISTRIBUTION" = "rpm" ]; then
+elif [ "$DISTRIBUTION" = "deb" ] || [ "$DISTRIBUTION" = "rpm" ]; then
     cp -va ../../../scripts/pkg/service_templates/opensearch/* "$OUTPUT/../"
     if [ "$MAJOR_VERSION" = "1" ]; then
         cp -va ../../../scripts/pkg/build_templates/legacy/opensearch/$DISTRIBUTION/* "$OUTPUT/../"
+    elif [ "$MAJOR_VERSION" = "2" ]; then
+        cp -va ../../../scripts/pkg/build_templates/current/opensearch/$DISTRIBUTION/* "$OUTPUT/../"
```

Comment:
Any circumstance that we need this? Seems like it's the same as below `else` condition.

Reply:
Yes, because 1.x will use build repo version of `legacy` and 2.x will use build repo version of `current`, while 3.x and above will maintain the folder structure while pulling the files directly from OS core.